1.7/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
   && wget -O /tmp/npm-3.10.3.zip https://github.com/npm/npm/archive/v3.10.3.zip \
   && wget -O /tmp/npm-3.10.10.zip https://github.com/npm/npm/archive/v3.10.10.zip \
   && wget -O /tmp/npm-5.0.3.zip https://github.com/npm/npm/archive/v5.0.3.zip \
+  && ln -s ../mods-available/xml2enc.load /etc/apache2/mods-enabled \
```

Comment:
Can you actually move this down as low as you can in the Dockerfile? It's nice to keep related stuff together, and that's what we have been doing up until now, but it radically increases build and push times.

Reply:
Ah never mind, the other file changes will invalidate the build cache anyway. Ignore this.